dapps/tests/app/test/another_storage_spec.js

```diff
@@ -42,6 +44,15 @@ contract("AnotherStorage", function() {
     assert.ok(parseInt(balance, 10) <= 5000000000000000000);
   });
 
+  it("should have the right balance for all other accounts ", async function() {
+    let balance;
```

Comment:
Since this is never reassigned, this probably should be `const`

Reply:
It is, look at line 51